contrib/stack/topsStack/stackSentinel.py

```diff
@@ -183,6 +183,9 @@ def createParser():
 
     parser.add_argument('-rmFilter', '--rmFilter', dest='rmFilter', action='store_true', default=False,
                         help='Make an extra unwrap file in which filtering effect is removed')
+    
+    parser.add_argument('-U', '--updateStack', dest='updateStack', action='store_true', default=False,
```

Comment:
Hi @bjmarfito . I think it might be ok to expose this parameter and force updating. But I don't see where your exposed parameter gets used?

Reply:
Thank you @hfattahi for the review. You are correct, I forgot to include the line of code so it can be used during processing. I will add it ASAP.